src/NetAnalyzers/Core/Microsoft.NetCore.Analyzers/Runtime/DetectPreviewFeatureAnalyzer.cs

```diff
@@ -626,6 +626,18 @@ private static bool SymbolIsAnnotatedAsPreview(ISymbol symbol, ConcurrentDiction
                     return true;
                 }
 
+                if (symbol.ContainingAssembly != null && symbol.ContainingAssembly.HasAttribute(previewFeatureAttribute))
```

Comment:
The docs here say that `symbol.ContainingAssembly` can be null if this symbol is part of multiple assemblies. I don't understand that case fully yet. Is this case common? If so, can someone give me a quick example of what this looks like?

Reply:
Question is not important anymore since I removed the call to Containing Assembly